network-api/networkapi/wagtailcustomization/static/wagtailadmin/css/custom-fix.css

```diff
@@ -17,49 +17,3 @@ body#wagtail
     height: unset;
   }
 }
-
-/*
-  To address bug from Wagtail v3
-  See context: https://github.com/mozilla/foundation.mozilla.org/issues/10194
```

Comment:
To test, 

- visit http://test.example.com:8000/cms/snippets/wagtailpages/buyersguideproductcategory/
- see if the language dropdown is now visible

Reply:
Working!